src/System.Security.Cryptography.Cng/ref/project.json

```diff
@@ -1,8 +1,8 @@
 {
   "dependencies": {
-    "System.Runtime": "4.0.0",
-    "System.IO": "4.0.0",
-    "System.Runtime.Handles": "4.0.0",
+    "System.Runtime": "4.1.0-rc3-24117-00",
```

Comment:
I can't think of any reason why this is bad?  It's not like those packages a) make this one have to become pre-release, or b) change the minimum framework of this package.

So this (and the rest of the change) LGTM.